test/functionalTest/cases/0999_update_single_attr_value_by_eid/update_single_attr_by_eid.test

```diff
@@ -30,6 +30,9 @@ brokerStart CB
 --SHELL--
 
 #
+# FIXME P6: steps 07, 09, 11, 13, and 15 in REGEXPECT of this test must change once the
```

Comment:
Probably we should close #2451 soon (I think it is quite clear no modification is needed, see my comment on that issue: https://github.com/telefonicaid/fiware-orion/issues/2451#issuecomment-242446354) thus removing these 3 lines before merging this PR.